api/config/routes.js

```diff
@@ -1,10 +1,13 @@
 const productRoutes = require('../routes/product');
 const userRoutes = require('../routes/user');
+const orderRoutes = require('../routes/order');
 
 module.exports = (app) => {
     app.use('/', productRoutes);
 
     app.use('/', userRoutes);
 
+    app.use('/', orderRoutes);
```

Comment:
You can say here /orders and don't write it on each route. Same for users and products 
```suggestion
    app.use('/orders, orderRoutes);
```